.travis.yml

```diff
@@ -0,0 +1,43 @@
+# Travis CI (MIT license) configuration file for Beans.
+# @link https://travis-ci.org/
+
+# For use with the Beans Theming Framework
+# @link https://github.com/Getbeans/Beans
+
+# This version does not include sniffers. Why? The codebase would fail.
+# Once we get the codebase compliant, then we'll update this file.
+
+sudo: false
+dist: trusty
+language: php
+
+env:
+  global:
+    - WP_VERSION=master WP_MULTISITE=0
+
+matrix:
+    fast_finish: true
+
+    include:
+        - php: 5.2
+          dist: precise
+        - php: 5.6
+        - php: 7.0
```

Comment:
There's probably not much point in testing 7.0 and 7.1.

`7.1` (or `7.2` as of a couple of weeks time) + nightly (with allowed failure for nightly) would probably be sufficient.

Reply:
@GaryJones Good point, changing to PHP 7.2.  I've also commented out 5.2.  Yes, the codebase is built on 5.2, but our tests run at 5.6.